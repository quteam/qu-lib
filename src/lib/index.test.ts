import {
    escape,
} from './';

test('Index test', () => {
    expect(escape).not.toBeNull();
});
