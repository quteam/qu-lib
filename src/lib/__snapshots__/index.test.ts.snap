// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`正确的导出模块 1`] = `
Array [
  "htmlEscape",
  "htmlUnescape",
  "dateToLang",
]
`;
